install/installer/pkg/components/usage/deployment.go

```diff
@@ -77,6 +79,8 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 							Image: ctx.ImageName(ctx.Config.Repository, Component, ctx.VersionManifest.Components.Usage.Version),
 							Args: []string{
 								"run",
+								"--schedule",
```

Comment:
My 2 cents, we should move this into a config file just like the public API. I'm happy to approve here to unblock but longer term we're better off with most config in files.

Reply:
You mean not have a `--schedule` flag at all but rather read the schedule from a config file?

I've tended to configure CLIs like this with multiple command line args, set via helm charts (or the installer here in the Gitpod case) in the past. What makes you prefer config files for this (not saying I disagree, just curious).

Reply:
Generally the pattern used for other components. I think it also somewhat simplifies configuration for the installer and in general observing what a component is configured with (but that's debatable).

Reply:
I think it's worth landing this is as is and align in subsequent PR.

Reply:
Made https://github.com/gitpod-io/gitpod/issues/10753 to track this.